internal/errcheck/errcheck.go

```diff
@@ -175,28 +172,18 @@ func (c *Checker) logf(msg string, args ...interface{}) {
 	}
 }
 
-func (c *Checker) load(paths ...string) (*loader.Program, error) {
-	ctx := build.Default
-	for _, tag := range c.Tags {
-		ctx.BuildTags = append(ctx.BuildTags, tag)
-	}
-	loadcfg := loader.Config{
-		Build: &ctx,
-	}
-
-	if c.WithoutGeneratedCode {
-		loadcfg.ParserMode = parser.ParseComments
-	}
+// loadPackages is used for testing.
+var loadPackages = func(cfg *packages.Config, paths ...string) ([]*packages.Package, error) {
```

Comment:
Can't you just do `var loadPackages = packages.Load` since the function signature is the same? And if so, is this variable needed?

Reply:
The variable is replaced by a different function in the unit tests. The definition could probably be simplified down as you propose.